### DISCUSSION TITLE
v1.3.14

### DISCUSSION THREAD

**Discussion by JaylyDev** [JaylyDev/terminator#8] [Announcements]
## New features
- Terminator now break blocks if there are 5 terminators or less in a 64 block area

## Changes
- Removed break block cooldown
- Terminator can do 1-3 damages to targeted mobs

## Bug fixes
- Terminator with no physics can now be only spawned with `/function terminator/nbt/nophysics`
  > Physics does not apply to terminator when not spawning with nophysics command in v1.3.13
- Players now cannot hit terminators that's spawned with `/function terminator/nbt/invulnerable`
- Fixed terminator's spawn event about merging "sequence" and "randomize" elements
  > ```
  > [Item][warning]-My World | actor_definitions |  entity:terminator | minecraft:entity | events | minecraft:entity_spawned | Item event has both 
  > randomize and sequence tags. Sequence will be ignored.
  > ```
  > **not sure why it says item, its deffo minecraft's problem lol**
- Fixed JSON errors on "terminator.json" file
  > ```
  > [AI][error]-My World | actor_definitions |  entities/terminator.json | parse error:
  >  * Line 2663, Column 1
  >    Missing ',' or '}' in object declaration
  > ```

## Installations:
This add-on is released for Minecraft version 1.17.30 or above, but there is a chance of it being compatible for Minecraft version 1.17.0

When you're installing this add-on in Minecraft 1.17.30, these are the experimentals options required to be activated.

![image](https://user-images.githubusercontent.com/65847850/137726804-3b01becd-1401-4bbc-863f-65a88aebe827.png)

> For versions that are below 1.17.30, you need to activate "Holiday Creator Features".

**Full Changelog**: https://github.com/JaylyDev/terminator/compare/v1.3.0...v1.3.14

<hr /><em>This discussion was created from the release <a href='https://github.com/JaylyDev/terminator/releases/tag/v1.3.14'>v1.3.14</a>.</em>

**Comment by bot174** [comment]
boo
